zipline/protocol.py

```diff
@@ -59,6 +61,10 @@
     'share_count',
 ]
 
+TRADING_DAYS_PER_YEAR = 252
+CVAR_LOOKBACK_DAYS = TRADING_DAYS_PER_YEAR * 2
+CVAR_CUTOFF = 0.05
```

Comment:
I would call this `DEFAULT_CVAR_CUTOFF` to emphasize that other values could be used.